build_config.rb

```diff
@@ -11,6 +11,7 @@ def gem_config(conf)
   conf.enable_bintest
   conf.enable_debug
   conf.enable_test
+  conf.enable_cxx_abi
```

Comment:
is there a way we can have this flag be autodetected instead of manually being added for anyone who wants to mruby-docopt?